fdbserver/workloads/RandomClogging.actor.cpp

```diff
@@ -62,49 +70,123 @@ struct RandomCloggingWorkload : TestWorkload {
 		g_simulator.clogInterface(machine->address.ip, t);
 	}
 
-	void clogRandomPair(double t) {
-		auto m1 = deterministicRandom()->randomChoice(g_simulator.getAllProcesses());
-		auto m2 = deterministicRandom()->randomChoice(g_simulator.getAllProcesses());
+	static void checkClogResult(Future<Void> res, WorkerInterface worker) {
+		if (res.isError()) {
+			auto err = res.getError();
+			if (err.code() == error_code_client_invalid_operation) {
+				TraceEvent(SevError, "ChaosDisabled")
+				    .detail("OnEndpoint", worker.waitFailure.getEndpoint().addresses.address.toString());
+			} else {
+				TraceEvent(SevError, "ClogginFailed")
```

Comment:
```suggestion
				TraceEvent(SevError, "CloggingFailed")
```
For better `grep`ability :)